### PR TITLE
chore: add Startup Module that can be used in other .NET plugins

### PR DESCRIPTION
**Pull request by joaoopereira**
on this PR we are creating a Startup Module class, that will be useful for other .NET plugins.
part of this code was spread through some plugins, and now is generalized on a single class.

### CODE REVIEW

File: cmf-cli/Constants/CliConstants.cs [hunk 1]
```diff
@@ -1,9 +1,26 @@
+using System.Management.Automation.Runspaces;
+using System.Security.Authentication.ExtendedProtection;
 using System.Text.RegularExpressions;
 
 namespace Cmf.CLI.Constants
 {
     public static class CliConstants
     {
+        /// <summary>
+        /// The package name used for INPMClient implementations
```
Comment (m-s-) [inline cmf-cli/Constants/CliConstants.cs:10]: There not much point saying where this is used (I have Find References), but what the actual use of these variables is

File: cmf-cli/Program.cs [hunk 2]
```diff
@@ -17,36 +17,16 @@ namespace Cmf.CLI
...
     public static class Program
     {
-
-       /// <summary>
-        /// program entry point
+        /// <summary>
+        /// Program entry point
         /// </summary>
-        /// <param name="args"></param>
+        /// <param name="args">The arguments.</param>
```
Comment (m-s-) [inline cmf-cli/Program.cs:23]: :P

File: cmf-cli/Constants/CliConstants.cs (hunk 1 above)
Comment (joaoopereira) [reply cmf-cli/Constants/CliConstants.cs]: I have removed these constants, doesn't make sense anymore

File: cmf-cli/Program.cs (hunk 2 above)
Comment (joaoopereira) [reply cmf-cli/Program.cs]: fixed 😅

**Review by m-s- — APPROVED** [review]

**Pull request merged**